test/unit/assorted/imports.test.ts

```diff
@@ -15,7 +15,7 @@ function* walk(root) {
   }
 }
 
-describe('importing mongodb driver', () => {
+describe.skip('importing mongodb driver', () => {
```

Comment:
```suggestion
describe('importing mongodb driver', () => {
```

Reply:
how did this get past the lint? 🤔

Reply:
What lint? there's nothing wrong with skipping tests